jvb/rootfs/defaults/jvb.conf

```diff
@@ -138,11 +138,22 @@ ice4j {
             }
             static-mappings = [
 {{ range $index, $element := $JVB_IPS -}}
+{{ if contains ":" $element -}}
```

Comment:
This breaks for IPv6. We could use # as the delimiter. A bit weird to read but would also work for IPv4 and IPv6.

Reply:
It should go without.

Reply:
Updated it, I know that invalid ip addresses are allowed with the provided regex but I think it is not part of the configuration to validate the addresses